Frameworks/Foundation/NSURLSession.mm

```diff
@@ -33,6 +33,21 @@
 
 #import <StubReturn.h>
 
+#define THROW_IF_NULL_REQUEST(request)                                         \
```

Comment:
i have the urge to say, since this macro is internal to this function it should have some signature indication that.